### PR TITLE
Add admin ckey to known alts panel

### PR DESCRIPTION
**Pull request by Mothblocks**
## About The Pull Request
Adds the admin who added the known alt to the known alts panel.

## Changelog

:cl:
admin: The admin who adds a known alt is now logged in the table.
/:cl:

### CODE REVIEW

File: SQL/database_changelog.txt [hunk 1]
```diff
@@ -1,13 +1,21 @@
 Any time you make a change to the schema files, remember to increment the database schema version. Generally increment the minor number, major should be reserved for significant changes to the schema. Both values go up to 255.
 
-The latest database version is 5.17; The query to update the schema revision table is:
+The latest database version is 5.19; The query to update the schema revision table is:
```
Comment (optimumtact) [inline SQL/database_changelog.txt:3]: wow bro wow

Comment (Mothblocks) [reply SQL/database_changelog.txt:3]: someone noticed forget who i think watermelon

**Comment by Burgermancoding** [comment]
the girl who wrote this

File: SQL/database_changelog.txt (hunk 1 above)
Comment (MrStonedOne) [reply SQL/database_changelog.txt:3]: there is another pr that edits the database version to 5.19, im gonna count this as 5.20

File: SQL/database_changelog.txt [hunk 2]
```diff
@@ -1,13 +1,21 @@
...
-INSERT INTO `SS13_schema_revision` (`major`, `minor`) VALUES (5, 18);
+INSERT INTO `SS13_schema_revision` (`major`, `minor`) VALUES (5, 19);
 
 In any query remember to add a prefix to the table names if you use one.
 
+Version 5.19, 11 November 2021, by Mothblocks
+Adds `admin_ckey` field to the `known_alts` table to track who added what.
+
+```
+ALTER TABLE `tgstation`.`known_alts`
```
Comment (MrStonedOne) [inline SQL/database_changelog.txt:15]:
```suggestion
ALTER TABLE `known_alts`
```

**Comment by Mothblocks** [comment]
Need to wait for Watermelon's PR to be merged x_x

**Comment by Mothblocks** [comment]
NOOO

**Comment by optimumtact** [comment]
lol

**Comment by Ghilker** [comment]
You called the horde

**Comment by LemonInTheDark** [comment]
I have been pinged

File: SQL/tgstation_schema.sql [hunk 3]
```diff
@@ -672,6 +672,7 @@ CREATE TABLE `known_alts` (
     `id` INT NOT NULL AUTO_INCREMENT,
     `ckey1` VARCHAR(32) NOT NULL,
     `ckey2` VARCHAR(32) NOT NULL,
+    `admin_ckey` VARCHAR(32) NOT NULL,
```
Comment (Jordie0608) [inline SQL/tgstation_schema.sql:675]:
```suggestion
    `admin_ckey` VARCHAR(32) NOT NULL DEFAULT '*no key*',
```

File: SQL/tgstation_schema_prefixed.sql [hunk 4]
```diff
@@ -672,6 +672,7 @@ CREATE TABLE `SS13_known_alts` (
     `id` INT NOT NULL AUTO_INCREMENT,
     `ckey1` VARCHAR(32) NOT NULL,
     `ckey2` VARCHAR(32) NOT NULL,
+    `admin_ckey` VARCHAR(32) NOT NULL,
```
Comment (Jordie0608) [inline SQL/tgstation_schema_prefixed.sql:675]:
```suggestion
    `admin_ckey` VARCHAR(32) NOT NULL DEFAULT '*no key*',
```

**Review by Jordie0608 — CHANGES_REQUESTED** [review]
aaaaaaaaaaaaaaaaaaa

**Pull request merged**